pkg/operator/ceph/csi/controller.go

```diff
@@ -58,6 +58,8 @@ type ReconcileCSI struct {
 	opManagerContext   context.Context
 	opConfig           opcontroller.OperatorConfig
 	clustersWithHolder []ClusterDetail
+	// the first cluster CR which will determine some settings for the csi driver
+	firstCephCluster *cephv1.ClusterSpec
```

Comment:
This is only needed temporarily until the csi operator is fully supported, but seemed simple enough to add for now until that transition is done.